eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/PostRestartHtlcCleaner.scala

```diff
@@ -71,8 +71,8 @@ class PostRestartHtlcCleaner(nodeParams: NodeParams, register: ActorRef, initial
       Metrics.PendingNotRelayed.update(brokenHtlcs.notRelayed.size)
       Metrics.PendingRelayedOut.update(brokenHtlcs.relayedOut.keySet.size)
       // Once we've loaded the channels and identified broken HTLCs, we let other components know they can proceed.
-      Try(initialized.map(_.success(Done)))
       context become main(brokenHtlcs)
+      sender ! Done
```

Comment:
Does this make sense? Since now PostRestartHtlcCleaner is initialized by a message it seems reasonable use ask/tell.